src/components/Checkout.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react"
+
+
+
+export default function Checkout({setCart}){
+    
+    const [form,setForm]=useState({firstName:""})
```

Comment:
Your default state should also include keys for the other HTML elements (lastName, email, zipcode). It works right now because I guess `undefined` acts the same as an empty string for the value attribute on the elements but we should, for best practices and so we don't run into weird bugs, set the default values of each input of type text to an empty string.
` const [form,setForm]=useState({
firstName:"",
lastName: "",
email: "",
zipcode: ""
})`